solution/quiz-4

```diff
@@ -0,0 +1,21 @@
+import csv
+data = [
+    ["2", "om", "4000"],
+    ["5", "sai", "4200"]
+]
+
+header = ["eid", "ename", "esalary"]
+with open("salary.csv", mode="w", newline="") as file:
+    writer = csv.writer(file)
+    writer.writerow(header)
+    writer.writerows(data)
```

Comment:
can you explain me ,what is the difference between writerow and writerows?

Reply:
writerow means single line and writerows means multiple lines